### ISSUE TITLE
[jvm-packages] xgboost4j-spark 0.80: Tree grow policy of lossguide 

### ISSUE THREAD

**Issue by David082** [dmlc/xgboost#3724]
Hi there,

1. May I treat the tree grow policy of "lossguide" as the same of leaf wise in Lightgbm?

2. As the document tells: 
grow_policy [default= depthwise]
    Controls a way new nodes are added to the tree.
    Currently supported only if tree_method is set to hist.
    Choices: depthwise, `lossguide
        depthwise: split at nodes closest to the root.
        lossguide: split at nodes with highest loss change.
So, does this means we can set grow_policy="lossguide" only when we set tree_method="hist" ?
recently, when set tree_method="hist" in xgboost4j-spark 0.80, encountered a problem: xgboost4j-spark does not support fast histogram for now.

**Comment by hcho3** [comment]
1. Yes
2. Yes. Unfortunately, 'hist' doea not support distributed training yet.

**Comment by cengjingmengxiang** [comment]
@hcho3 ,does distributed verson only support tree_method=approx?

**Comment by David082** [comment]
@hcho3 Thanks, and looking forward to have lossguide in distributed version!
